homeassistant/components/solaredge/const.py

```diff
@@ -65,4 +66,15 @@
     "solar_power": ["PV", "Solar Power", None, "mdi:solar-power", False],
     "grid_power": ["GRID", "Grid Power", None, "mdi:power-plug", False],
     "storage_power": ["STORAGE", "Storage Power", None, "mdi:car-battery", False],
+    "purchased_power": ["Purchased", "Imported Power", None, "mdi:flash", False],
+    "production_power": ["Production", "Production Power", None, "mdi:flash", False],
+    "consumption_power": ["Consumption", "Cosumption Power", None, "mdi:flash", False],
```

Comment:
Typo:
"Consumption Power"

Reply:
Thanks, I've fixed and pushed but PR is closed now sorry.